10/umbraco-cms/tutorials/creating-a-property-editor/part-4.md

```diff
@@ -83,12 +79,13 @@ public IEnumerable<Person> GetAll()
 }
 ```
 
-Inside the `GetAll()` method, we now write a bit of code, that connects to the database, creates a query and returns the data, mapped to the `Person` class above:
+Inside the `GetAll()` method, we write a bit of code. The code connects to the database, creates a query, and returns the data, mapped to the `Person` class above:
 
 ```csharp
-private readonly IScopeProvider scopeProvider;
+private readonly Umbraco.Cms.Infrastructure.Scoping.IScopeProvider scopeProvider;
+
 
-public PersonApiController(IScopeProvider scopeProvider)
+public PersonApiController(Umbraco.Cms.Infrastructure.Scoping.IScopeProvider scopeProvider)
```

Comment:
instead of adding twice Umbraco.Cms.Infrastructure.Scoping when defining the class, it can be added only once as a namespace: using Umbraco.Cms.Infrastructure.Scoping;

Maybe this might be better and also best practice instead of adding a namespace inside a class?
![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/96102434-419e-4bb6-93ca-dc2556ac7f9b)

Reply:
That does seem more correct than whatever it is I am doing 😅